Add basic ref docs for run types (#3792)
langchain-ai/docs: src/langsmith/trace-query-syntax.mdx

| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| `project_id` / `project_name` | The project(s) to fetch runs from - can be a single project or a list of projects.                                                                                                                                             |
| `trace_id`                    | Fetch runs that are part of a specific trace.                                                                                                                                                                                  |

> | `run_type`                    | The [type of run](/langsmith/run-data-format#run-types) to get, such as `llm`, `chain`, `tool`, `retriever`, etc.                                                                                                                              |

Reviewer: ```suggestion
| `run_type`                    | The [type of run](/langsmith/run-data-format#run-types) to get, for example, `llm`, `chain`, `tool`, `retriever`.                                                                                                                              |
```
